lib/command.h

```diff
@@ -82,7 +82,6 @@ enum node_type {
 	AUTH_ENABLE_NODE,	/* Authentication mode for change enable. */
 	ENABLE_NODE,		 /* Enable node. */
 	CONFIG_NODE,		 /* Config node. Default mode of config file. */
-	DEBUG_NODE,		 /* Debug node. */
```

Comment:
This change is here to "fix" the order in config write?

This is a little problematic since `debug XYZ` commands should be processed before other config on startup, so you get debug messages while things are getting configured/started.

⇒ move `PREFIX_IPV6_NODE` / `PREFIX_NODE` above `DEBUG_NODE` (which stays where it is) instead

Reply:
Thanks, fixed.